terraform/tfvars/production.tfvars

```diff
@@ -0,0 +1,152 @@
+# Organization admins
+admins = [
+  "tim-schilling",
+  "williln",
+  "ryancheley",
+  "Stormheg",
+  "cunla",
+]
+# Organization members
+members = [
+  "gav-fyi",
+  "jcjudkins",
+  "joshuadavidthomas",
+  "matthiask",
+  "nanorepublica",
+  "Natim",
+  "pfouque",
+  "priyapahwa",
+  "testSchilling",
+]
+
+organization_secrets = {
+  #   "GPG_PASSPHRASE" = {
+  #     description = "GPG Passphrase used to encrypt plan.out files"
+  #     visibility  = "all"
+  #   }
+}
+
+# Organization repositories
+repositories = {
+  # Keep the following repositories in alphabetical order
+
+  ".github" = {
+    description              = "A Special Repository."
+    enable_branch_protection = false
+
+    topics = []
+    push_allowances = []
+  }
+
+  "controls" = {
+    description              = "The controls for managing Django Commons projects"
+    enable_branch_protection = false
+
+    topics = []
+    push_allowances = []
+    visibility = "public"
+  }
+
+  "membership" = {
+    description = "Membership repository for the django-commons organization."
+    visibility  = "public"
+    topics = []
+    push_allowances = []
+  }
+
+  "django-commons-playground" = {
+    description = "A sample project to test things out"
+    topics = []
+  }
+
+}
+
+team_parents = {
+  "Admins" = {
+    description = "django-commons administrators"
+    maintainers = [
+      "tim-schilling",
+      "williln",
+      "ryancheley",
+      "Stormheg",
+      "cunla",
+    ]
+    members = null
+  }
+
+  "django-community-playground" = {
+    description = "django-community-playground team"
+    maintainers = [
+      "tim-schilling",
+      "williln",
+      "ryancheley",
+      "Stormheg",
+      "cunla",
+    ]
```

Comment:
I don't think we should set this for the teams. With the org owner role that we all have, we can adjust any team that's needed.

Reply:
This is a bit limiting.
When adding a member, one needs to set whether they are member/maintainer.
So if we do it via terraform, we need to set permissions. Otherwise, when one changes it manually, terraform will change it back in the next change.

We can simply say that org owners will change/approve/merge PRs and control permissions through that.

Reply:
I think we're saying the same thing. Changes to teams should occur via a PR. Given that, we shouldn't need to set the `maintainers` on the repository teams.

Reply:
Oh, I hear you. That makes sense.
We can remove the team maintainers part.

Reply:
@cunla I'm going to remove these in a commit. Feel free to revert it if you found it was necessary.